pkg/apis/velero/v1/backup.go

```diff
@@ -82,6 +82,12 @@ type BackupSpec struct {
 	// VolumeSnapshotLocations is a list containing names of VolumeSnapshotLocations associated with this backup.
 	// +optional
 	VolumeSnapshotLocations []string `json:"volumeSnapshotLocations,omitempty"`
+
+	// DefaultRestic specifies whether restic should be used to take
+	// backup of all pod volumes by default.
+	// +optional
+	// + nullable
+	DefaultRestic *bool `json:"defaultRestic,omitempty"`
```

Comment:
I'm trying to think of a more descriptive name for this field that's also not a full sentence, and failing...I'll keep thinking

Reply:
Maybe `ResticBackupVolumesByDefault`? A little longer but also I think more descriptive? Open to other suggestions too.

Reply:
@nrb @carlisia WDYT on naming?

Reply:
I spent like20m thinking of a better name and failed.
Suggestions welcome :)

Reply:
`DefaultVolumesToRestic`?

Reply:
works for me

Reply:
`DefaultVolumesToRestic` is the winner